opennms-base-assembly/src/main/filtered/etc/datacollection/mib2-coffee.xml

```diff
@@ -0,0 +1,13 @@
+<datacollection-group xmlns="http://xmlns.opennms.org/xsd/config/datacollection" name="MIB2-Coffee">
+   <group name="mib2-coffee-rfc2325" ifType="ignore">
+      <mibObj oid=".1.3.6.1.2.1.10.132.2" instance="0" alias="coffeePotCapacity" type="integer"/>
+      <mibObj oid=".1.3.6.1.2.1.10.132.4.1.2" instance="0" alias="coffeePotLevel" type="integer"/>
+      <mibObj oid=".1.3.6.1.2.1.10.132.4.1.6" instance="0" alias="coffeePotTemp" type="integer"/>
+   </group>
+   <systemDef name="Enterprise MIB2-Coffee">
+      <sysoidMask>.1.3.6.1.4.1.</sysoidMask>
+      <collect>
+         <includeGroup>mib2-coffee-rfc2325</includeGroup>
```

Comment:
Does it make sense to include this by default ? Doesn't look like the kind of things that ALL devices should support.